include/swift/Serialization/ModuleFormat.h

```diff
@@ -1662,6 +1664,7 @@ namespace decls_block {
   >;
 
   // SWIFT_ENABLE_TENSORFLOW
+  // TODO: why didn't I add a bit for linear flag here?
```

Comment:
Forgot to do this in previous PR, will add it in this PR and add corresponding tests.